### DISCUSSION TITLE
Login using LN wallet

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2004] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
Use [lnurl-auth](https://xn--57h.bigsun.xyz/lnurl-auth.html) for authentication

**Describe the solution you'd like**
Use https://github.com/Horndev/lnurl.net to integrate lnurl-auth in btcpayserver

**Sketch/Image/Wireframe/Mockup**
 
![20201021_153754](https://user-images.githubusercontent.com/13405205/96705922-8799a800-13b3-11eb-9995-c7ac5474d699.jpg)

**Describe alternatives you've considered**
Use email and password for login

**Additional context**

![Screenshot_20201021-154553__01](https://user-images.githubusercontent.com/13405205/96706958-e14ea200-13b4-11eb-8f5d-2ecf8e59f696.jpg)

![Screenshot_20201021-154609__01](https://user-images.githubusercontent.com/13405205/96706979-e9a6dd00-13b4-11eb-8009-4874333b002b.jpg)

_Screenshots are from https://lnmarkets.com and https://etleneum.com_

**Comment by pavlenex** [comment]
Related #1692

**Comment by pavlenex** [comment]
~~I'll close this one as a duplicate of #1692~~

mhm, actually it's not the same, I misread, my bad. Re-opened
